ios/RohTVApp-tvOS/Info.plist

```diff
@@ -35,6 +35,20 @@
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string></string>
+	<key>UIAppFonts</key>
```

Comment:
Can we have a one line comment to explain what this is for?

Reply:
This is the key for the fonts to be embedded in an iOS/tvOS app. (There is a process to add them automatically, but it doesn't work for tvOS only iOS, so I added manually).

Reply:
Cool, that's the one-liner you need :)

Reply:
Added!